chartfx-math/src/main/codegen/de/gsi/math/MathBaseGen.java

```diff
@@ -0,0 +1,431 @@
+/*************************************************************************
+ * Originally Based on CERN's ROOT analysis frame work:
+ * 
+ * @see root.cern.ch for details Copyright (C) 1995-2004, Rene Brun and Fons Rademakers. Authors: Rene Brun, Anna
+ *      Kreshuk, Eddy Offermann, Fons Rademakers All rights reserved. Java port and extension by: Ralph J. Steinhagen,
+ *      CERN, BE-BI, 2009 For the licensing terms see LICENSE. For the list of contributors see $ROOTSYS/README/CREDITS
+ *************************************************************************/
+
+package de.gsi.math;
+@SuppressWarnings("PMD.UnnecessaryFullyQualifiedName") // the fully qualified name is needed, because we have our own Math class
+public class MathBaseGen {
```

Comment:
Codacy found an issue: [The utility class name 'MathBaseGen' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)